src/operations/revoke_sponsorship.js

```diff
@@ -195,6 +195,42 @@ export function revokeClaimableBalanceSponsorship(opts = {}) {
   return new xdr.Operation(opAttributes);
 }
 
+/**
+ * Creates a "revoke sponsorship" operation for a liquidity pool.
+ *
+ * @function
+ * @alias Operation.revokeLiquidityPoolSponsorship
+ * @param {object} opts – Options object.
+ * @param {string} opts.liquidityPoolId - The sponsored liquidity pool ID in 'hex' string.
+ * @param {string} [opts.source] - The source account for the operation. Defaults to the transaction's source account.
+ * @returns {xdr.Operation} xdr Operation.
+ *
+ * @example
+ * const op = Operation.revokeLiquidityPoolSponsorship({
+ *   liquidityPoolId: 'dd7b1ab831c273310ddbec6f97870aa83c2fbd78ce22aded37ecbf4f3380fac7',
+ * });
+ *
+ */
+export function revokeLiquidityPoolSponsorship(opts = {}) {
```

Comment:
Might be a bit surprising to have a separate operation here that isn't mirrored in the actual operations? Not sure, tbh...